en/components/overlay.md

```diff
@@ -110,12 +110,18 @@ The Overlay Service's [`attach()`]({environment:angularApiUrl}/classes/igxoverla
   - `attach(component, settings?, moduleRef?)`
 
 The first parameter in both overloads is mandatory and represents the content that will be shown in the overlay. There are a couple of different scenarios how the content can be passed:
-  - A component definition - When passing a component in as the first argument, the overlay service creates a new instance of that component and dynamically attaches it to the `overlay` DOM. If `moduleRef` is provided the service will use the module's `ComponentFactoryResolver` and `Injector` when creating the `ComponentRef` instead of the root ones.
-  - An `ElementRef` to an existing DOM element (illustrated in the sample above) - Any view that is already rendered on the page can be passed through the overlay service and be rendered in the overlay DOM. Using this approach when [`show(id)`]({environment:angularApiUrl}/classes/igxoverlayservice.html#show) is called, overlay will:
-    - Get the reference to the passed view from Angular 
-    - Detach the view from the DOM and leave an anchor in its place
-    - Re-attach the view to the overlay, using the [`show()`]({environment:angularApiUrl}/classes/igxoverlayservice.html#show) method settings or falling back to the default overlay settings
-    - On close, it will re-attach the view back to it's original location in the DOM
+  - A component definition - When passing a component in as the first argument, the overlay service creates a new instance of that component and dynamically attaches it's `ElementRef` to the `overlay` DOM. If `moduleRef` is provided the service will use the module's `ComponentFactoryResolver` and `Injector` when creating the `ComponentRef` instead of the root ones.
+  - An `ElementRef` to an existing DOM element (illustrated in the sample above) - Any view that is already rendered on the page can be passed through the overlay service and be rendered in the overlay DOM.
+
+In both cases [`attach()`]({environment:angularApiUrl}/classes/igxoverlayservice.html#attach) method will:
+  - Get the reference to the passed view from Angular
+  - Detach the view from the DOM and leave an anchor in its place
+  - Re-attach the view to the overlay using provided [`OverlaySettings`]({environment:angularApiUrl}/interfaces/overlaysettings.html) or falling back to the default overlay ones
+
+Calling then [`show(id)`]({environment:angularApiUrl}/classes/igxoverlayservice.html#show) will play the open animation if any and will show the attached content. Calling [`hide(id)`]({environment:angularApiUrl}/classes/igxoverlayservice.html#hide) will play close animation in any and will hide the attached content.
```

Comment:
Calling then [`show(id)`]({environment:angularApiUrl}/classes/igxoverlayservice.html#show) will play the open animation **if any** (if there is any?) and will show the attached content.

Calling [`hide(id)`]({environment:angularApiUrl}/classes/igxoverlayservice.html#hide) will play close animation **in any** (in any what?) and will hide the attached content.